src/reanimated2/mock.ts

```diff
@@ -12,6 +12,10 @@ const ID = (t) => t;
 const IMMEDIATE_CB_INVOCATION = (cb: () => unknown) => cb();
 
 class BaseAnimationMock {
+  createInstance() {
+    return this;
+  }
```

Comment:
since this is also exposed I added it. 
However `createInstance` might only makes sense internally inside the library(from my perspective, correct me if I am wrong). I can also delete it if it does not makes sense.